Fix power select styles

alex-ju opened the pull request:
### :pushpin: Summary

This PR brings the `<PowerSelect>` style overrides to work on the [new website](https://hds-website-hashicorp.vercel.app/overrides/power-select) as they did on the [old one](https://design-system-components-hashicorp.vercel.app/overrides/power-select).

### :hammer_and_wrench: Detailed description

For an unknown reason, on the old website, we were benefitting from a series of styles that are no longer applied to the new website. I have checked the configuration and the only difference between the two is the following (extract from `ember-cli-build`)

```
    'ember-power-select': {
      theme: false,
    },
```

Applying the same configuration to the new website doesn't change anything visually, hence the only option I could see was to port the missing CSS declarations into the override to make the overrides more resilient.

I'm still scratching my head around this, but not sure spending more time on would be of help.

### :camera_flash: Screenshots

There are many subtle changes, but the most obvious one is in the multiselect version.

<table>
<tr><th>Before</th><th>After</th></tr>
<tr><td>

![hds-website-hashicorp vercel app_overrides_power-select](https://user-images.githubusercontent.com/788096/211341856-c0473422-a64b-4704-a2c3-d196cab92fce.png)


</td><td>

![localhost_4200_overrides_power-select](https://user-images.githubusercontent.com/788096/211341875-371dfc39-1eae-4218-9734-6bdf1de9744a.png)


</td></tr>
</table>

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

Dhaulagiri commented:
> For an unknown reason, on the old website, we were benefitting from a series of styles

@alex-ju is [this line](https://github.com/hashicorp/design-system/blob/e93e8a603139c12cd92bd29536007e2c177118de/packages/components/ember-cli-build.js#L12) what is making the magic happen on the old site?

alex-ju commented:
> > For an unknown reason, on the old website, we were benefitting from a series of styles
> 
> @alex-ju is [this line](https://github.com/hashicorp/design-system/blob/e93e8a603139c12cd92bd29536007e2c177118de/packages/components/ember-cli-build.js#L12) what is making the magic happen on the old site?

I thought about that too, but adding it (with the updated path) doesn't change things 🤔

alex-ju commented:
> > > For an unknown reason, on the old website, we were benefitting from a series of styles
> > 
> > 
> > @alex-ju is [this line](https://github.com/hashicorp/design-system/blob/e93e8a603139c12cd92bd29536007e2c177118de/packages/components/ember-cli-build.js#L12) what is making the magic happen on the old site?
> 
> I thought about that too, but adding it (with the updated path) doesn't change things 🤔

It turns out I wasn't following my own instructions – which is follow the [Ember Power Select installation guide](https://ember-power-select.com/docs/installation) so I forgot to import the styles 🤦 

I dropped the first commit with CSS adjustments and added a new one with the import.

Dhaulagiri commented:
Nice!
